models/veiculo/sppo_veiculo_dia.sql

```diff
@@ -10,31 +26,28 @@
 WITH
   licenciamento AS (
   SELECT
-    DATE("{{ var('run_date') }}") AS data,
+    data,
     id_veiculo,
     placa,
     tipo_veiculo,
     indicador_ar_condicionado,
-    TRUE AS indicador_licenciado
+    TRUE AS indicador_licenciado,
+    CASE
+      WHEN DATE("{{ var('run_date') }}") < "2024-03-01" THEN TRUE
```

Comment:
```suggestion
      WHEN DATE("{{ var('run_date') }}") < "2024-03-01" THEN NULL
```

@eng-rodrigocunha antes não existia essa classificação